src/components/containers/visibility.tsx

```diff
@@ -86,6 +86,8 @@ export const getCountryRule = () => {
     const is_eu_uk = !(!is_eu && !is_uk)
     const is_row = !is_eu_uk
 
+    console.log({ is_eu, is_uk, is_non_uk, is_non_eu, is_eu_uk, is_row })
```

Comment:
Please remove this console.log
it will affect production performance

Reply:
Thanks for the feedback @niloo-fs  we will remove the console after we solve the issue on staging

Reply:
Anytime